README.md

```diff
@@ -6,5 +6,21 @@ This project is **WIP**:
 
 ▮▯▯▯▯▯▯▯▯▯▯▯▯ 8% (17/208 tests)
 
+## Hot to implement a test
+
+1. Download `https://raw.githubusercontent.com/jorgicor/bas55/master/tests/PXXX.{BAS, ok, eok}` to
```

Comment:
What do you think about adding the wget command?

Reply:
Hard to specify a suffix. Some people prefer curl. :)